porting/device-configuration.rst

```diff
@@ -0,0 +1,78 @@
+Device configuration
+--------------------
+
+The rootfs (root file system) you have installed on your device contains a number of configuration files with some standard settings that do not necessarily conform to your device. These must be modified to fit your device, but you are not allowed to do so by mounting the rootfs read/write and editing them directly on the device. Instead, working on your host pc, you edit the files there and inject them into your system at build time as outlined below.
```

Comment:
"you are not allowed" sounds a bit arbitrary. there is a reason for it. How about something like

 The rootfs (root file system) you have installed on your device contains a number of configuration files with some standard settings that do not necessarily conform to your device. These must be modified to fit your device. The rootfs itself is read only and identical for all devices. Below it is described how to adjust the configuration with overlay files.

Reply:
Resolved, but in new branch. New PR will be submitted.